axelar-chains-config/info/mainnet.json

```diff
@@ -2007,6 +2007,43 @@
         "url": "https://explorer.immutable.com",
         "api": "https://explorer.immutable.com/api"
       }
+    },
+    "fraxtal": {
+      "name": "Fraxtal",
+      "id": "fraxtal",
+      "axelarId": "fraxtal",
+      "chainId": 252,
+      "rpc": "https://rpc.frax.com",
+      "tokenSymbol": "frxETH",
+      "confirmations": 2,
```

Comment:
i changed this from 1 to 2 as with other L2 chains. not sure if  1 was intentional

Reply:
2 is good just to confirm there's no immediate reorg